ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/AffectedImagesTable.tsx

```diff
@@ -132,6 +147,9 @@ function AffectedImagesTable({ images, getSortParams, isFiltered }: AffectedImag
                                     )}
                                 </span>
                             </Td>
+                            <Td dataLabel="CVSS">
+                                {cvss.toFixed(1)} ({scoreVersion})
```

Comment:
That's a good point. There are some places though that `cvss` is pulled in and used directly from a gql response, so I'll move the `toFixed(1)` call to a reusable component so that we can still encapsulate it.